interface/src/app/about/about.tpl.html

```diff
@@ -1,8 +1,21 @@
-<div>
-  <h1 class="page-header">
-    About ADAGE
-  </h1>
-  <p>
-    This is where some explanatory text about ADAGE should go.
-  </p>
+<div class="page-header">
+  <h1>About ADAGE</h1>
 </div>
+<h3>
```

Comment:
This `h3` tag is causing all of the text to appear bold, as if the page were yelling. Can we reserve these tags for actual headers (brief titles) only? I recommend simple `<p>` tags instead.